chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -71,6 +70,11 @@ type BitcoindNotifier struct {
 
 	blockEpochClients map[uint64]*blockEpochRegistration
 
+	// hintCache is a cache used to query and update the latest height hints
+	// for a transaction or outpoint. Each height hint represents the exact
+	// height at which a transaction was confirmed or an outpoint was spent.
```

Comment:
I guess this is not exactly correct... It should be the "earliest height at which..."?